client-runtime/src/main/java/com/microsoft/rest/v3/annotations/GET.java

```diff
@@ -14,19 +14,20 @@
 /**
  * HTTP GET method annotation describing the parametrized relative path to a REST endpoint for resource retrieval.
  *
- * The required value can be either a relative path or an absolute path. When it's
- * an absolute path, it must start with a protocol or a parametrized segment.
- * (Otherwise the parse cannot tell if it's absolute or relative)
+ * <p>The required value can be either a relative path or an absolute path. When it's an absolute path, it must start
+ * with a protocol or a parametrized segment (otherwise the parse cannot tell if it's absolute or relative).</p>
```

Comment:
nit: spelling. parameterized.

Reply:
Thanks, will fix. To be fair (due to your comment below thanking me), I didn't write any of the docs, I just geeked out and formatted them so they don't appear in the javadoc output in a massive blob of text.

Reply:
Ahh. Gotcha. :) I have a new appreciation for javadocs because of this.. it made it really easy to reason about the public surface area.

Reply:
Yeah, you don't want to review the javadoc output of this repo before this PR is merged....it is just one big paragraph of text. Also, just committed a fix to all typos of parameterised....I mean, parameterized :-)